docs-src/docs/reference/task_definition.md

```diff
@@ -31,10 +31,10 @@ The following snippet contains the task definition for the [Guestbook](../guides
 * **Name** The name of the container
 
 !!! warning
-If you wish to update your task definition, the container names **must** remain the same.
-If any container names are changed or removed in an updated task definition,
-ECS will not know how the existing container(s) should be mapped over and you will not be able to deploy the updated task definition.
-If you encounter a scenario where you must change or remove a container's name in a task definition, we recommend re-creating the Layer0 Deploy and Service.
+    If you wish to update your task definition, the container names **must** remain the same.
```

Comment:
Are these spaces consistent with other parts of the docs? Do we need to update elsewhere?

Reply:
For any of the `!!!warning`-and-similar blocks, any text that should be included in the [admonition](https://squidfunk.github.io/mkdocs-material/extensions/admonition/) needs to be indented by four spaces, so this spacing should be consistent with any other not-broken admonition blocks.

Reply:
Thanks! I'll look back through and see if there are any other broken ones.

Reply:
:+1: Awesome, thanks. I'm going through and doing the same, so it'll be good to have another set of eyes.